### ISSUE TITLE
(question, not bug): best way to chage sorting of note name and directory

### ISSUE THREAD

**Issue by rdiaz02** [org-noter/org-noter#99]
This is a question, not a bug.

I use vertico. When creating a note, the order of candidates for the questions "What name do you want" and "Where do you want to save it" were, most of the time, not what I wanted (what I wanted were often at the end of the candidates).  The cause is my default `vertico-sort-function`:  I checked that on adding a `(vertico-sort-function nil)` in 
 https://github.com/org-noter/org-noter/blob/691efc3ed4a2828d791a148e53851365c2eb380f/org-noter.el#L216 I get candidates in the order I often want.

What I am doing now is to `advice-add :around`: 

```elisp
(defun rdu-vertico-sort-function-nil-advice (orig-fun &rest args)
      "Set vertico-sort-function to nil while calling the the org-noter--create-session-from-document-file-default"
      (let* ((current-vertico-sort-fun vertico-sort-function)
  	     (vertico-sort-function nil))
  	(apply orig-fun args)
  	(setq vertico-sort-function current-vertico-sort-fun)))
(advice-add 'org-noter--create-session-from-document-file-default
		:around #'rdu-vertico-sort-function-nil-advice)
```

Is this sensible, or are there better ways?

**Comment by dmitrym0** [comment]
I use vertico/orderless  as well. I always narrow the options down though, so I haven't observed the behaviour you're seeing.  My `vertico-sort-function` is set to `vertico-sort-history-length-alpha`.

Anyhow, I dont see any problems with the advice if it works for you.

**Comment by rdiaz02** [comment]
Thanks! I use `prescient-completion-sort` and my preferred candidates (which also happen to be the longest strings) were often the last among a set of about four. By setting  `vertico-sort-function` to  nil they show up as the first one (I noticed there is a very handy reverse of the list of candidates inside `org-noter--create-session-from-document-file-default`) , so I only need to hit "enter".

Closing the issue. Thanks again.
